Fixing a broken unit test.

raphtheb opened the pull request:
This test isn't passing properly. Looking at the git log,
it seems like it was a typo/mistake when open-sourcing
this project which has gone undiscovered until now.

This is a 1-word change, typing down this long a commit
message feels truly overkill.

porcupie commented:
From what I see, the Topic is "sample_error_log" and not "sample-error.log" - not sure how you can get the test to pass with your changes?

raphtheb commented:
Ha! This is what happens when you think "Sure, i can do one more PR at 1 in the morning". Sorry about that! I'm amending the commit immediately to be "sample_error_log"

porcupie commented:
No worries :) Thanks

rtheberge committed:
Fix a broken unit test

This time using the correct value of sample_error_log.

This test isn't passing properly. Looking at the git log,
it seems like it was a typo/mistake when open-sourcing
this project which has gone undiscovered until now.

This is a 1-word change, typing down this long a commit
message feels truly overkill.

raphtheb commented:
There, that did it.

porcupie commented:
LGTM - Thanks @raphtheb 
@imrangit PTAL ...
